### PR TITLE
Change top-level README to use dynamically generated lists

### PR DESCRIPTION
**Pull request by rphair**
Last step in implementing #883.

Before reviewing, please read (and perhaps comment on) description of the PR state tagging vocabulary that was applied through the whole PR queue this week: https://github.com/cardano-foundation/CIPs/wiki/301.-State-tagging

1 - The lists of candidate CIPs and CPSs can no longer be sorted by the candidate CIP number as they were in the manually generated tables.  Interestingly, [GitHub cannot do this](https://github.com/orgs/community/discussions/43151).  This is a toss-up: maybe before it was easier to find a particular proposal & now it's easiest to find the oldest or newest proposals (plus the user can change to a different sort ordering if they want, add more search terms, etc.).

2 - If any wrong items show up on these dynamically generated lists, it means somehow a preposterous set of tags has been applied.  Some care is applied to the queries to prevent this in many cases (i.e. filtering out any "administrative" tagged proposals, even though they generally are not supposed to have a "state" in the review process).

3 - The deleted lists had many likely-abandoned proposals even after I made regular cullings of closed proposals.  The list of "actively reviewed" CIP/CPS PRs is actually quite small & manageable.  Likewise the total PR queue will be rather small once the "Stalled" proposals are either closed _or_ the current & ongoing attention causes the authors to revive and progress them (I will be happy either way).

---

([updated README rendered in branch: changes begin here](https://github.com/rphair/CIPs/tree/replace-tables-with-state-queries#proposals-under-review-cip))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,33 +150,21 @@ Below are listed tentative CIPs still under discussion with the community. They
...
+The following link lists "candidate" CPSs still under discussion with the community; these are assigned numbers to avoid later clashes and to facilitate community discussion (see further below for stalled proposals):
 
-<p align="right"><i>Last updated on 2024-07-27</i></p>
+**[CPS pull requests under active review](https://github.com/cardano-foundation/CIPs/pulls?q=is%3Apr+is%3Aopen+%2F%5ECPS-%2F+in%3Atitle+label%3A%22State%3A+Confirmed%22%2C%22State%3A+Last+Check%22+draft%3Afalse+-label%3AUpdate%2CCorrection%2CTranslation%2C%22Bi-Weekly+Notes+%2F+Editorial+Housekeeping%22%2C%22CIP-0010%3A+new+registry+entry%22%2C%22CIP-0067%3A+new+label%22%2C%22CIP-0088%3A+new+extension%22)**
 
 ## Stalled / Waiting For Authors
 
-The following list contains proposals that have been under review and for which actions are now awaiting updates from their original authors. Proposals stalled for several months without any updates from their authors and will eventually be closed. However, authors are invited to re-open pull requests or open new ones should they want to bring the discussion back to life.
+The following links list proposals deemed ready for review but requiring further update
```
Comment (Ryun1) [inline README.md:159]:
```suggestion
The following links list proposals deemed ready for review but requiring further update.
```
punctuation.

**Review by Ryun1 — APPROVED** [review]
I think this a awesome change
Providing more clarity to proposers AND reducing editor overhead 🤝

> description of the PR state tagging vocabulary that was applied through the whole PR queue this week: https://github.com/cardano-foundation/CIPs/wiki/301.-State-tagging

This is pretty damn good, I have very little to be able to add on this

> The lists of candidate CIPs and CPSs can no longer be sorted by the candidate CIP number as they were in the manually generated tables

I feel like this is fine, considering the benefits that using these state labels brings

File: README.md (hunk 1 above)
Comment (rphair) [reply README.md:159]: @Ryun1 keep reading down to line 160... the sentence & paragraph continue to form the following text (as seen [here in branch](https://github.com/rphair/CIPs/tree/replace-tables-with-state-queries?tab=readme-ov-file#stalled--waiting-for-authors)):
> The following links list proposals deemed ready for review but requiring further update from the original author(s) or other confirmation of proposal elgibility (if considered deprecated):

**Review by Crypto2099 — APPROVED** [review]
Very good idea!

**Comment by rphair** [comment]
One last idea (for a future PR, since we should agree on the language how it's presented): I believe it would help to have a link to the `Update` proposals before the "Stalled" section, since people who follow the body of CIPs would be interested in following that.

**Pull request merged**